public/js/actions/pause.js

```diff
@@ -206,6 +206,15 @@ function addExpression(expression) {
   };
 }
 
+function setExpressionInputVisibility(visibility) {
```

Comment:
Hmm, at this point we're opting to use component state local UI data. Some examples of this are, the search bar value, app search being shown, ... 

I think at some point, we might want to start using redux state more for UI data like this, but for the moment it is simpler to use component state.